### DISCUSSION TITLE
How should we configure it to ignore all built-in transforms of swc (only execute our plugin-specific transforms)

### DISCUSSION THREAD

**Discussion by Nirvana-Jie** [swc-project/swc#9516] [Q&A]
In some scenarios, we use swc's rust plugin to complete a specific transform, but the built-in transform will cause more time consuming

**Comment by kdy1** [comment]
There's `jsc.experimental.disableBuiltinTransformsForInternalTesting` but it may be removed in the future without a notice.

https://github.com/swc-project/swc/blob/d0dd53137a87dba85e6bca9d74cc682da7057289/packages/types/index.ts#L668

**Reply by Nirvana-Jie** [reply]
Thanks！It works
